Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -267,12 +267,14 @@ def SetPlatformEnv(self):
         self.env.SetValue("DELTA_CONF_POLICY", self.mws.join(self.ws, "QemuQ35Pkg", "CfgData", "Profile1.dlt") + ";" +\

```

Comment:
The line above this, should we remove them?

Reply:
Yes, part of the cleanup I will do.